Library/Homebrew/language/node.rb

```diff
@@ -32,13 +44,13 @@ def self.setup_npm_environment
       end
     end
 
-    def self.std_npm_install_args(libexec)
+    def self.std_npm_install_args(libexec, prepublish_requires_deps = false, pack_skip_prepublish = false)
```

Comment:
Something like `requires_deps` would be strictly speaking incorrect, because the deps would be required in any case on install time. Having to set `requires_deps: false` to not install deps at pack/prepublish time, but have them installed in the install step would likely create more confusion than the longer names.